collections/_article/collaboration-with-google-forge-2023.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Announcing a collaboration with Google and The Forge"
+excerpt: "We are excited to announce that we have partnered with Google and The Forge to bring some helpful performance optimizations to our Vulkan mobile backend."
+categories: ["news"]
+author: Clay John
+image: /storage/blog/covers/godot-foundation-update-2023.jpg
+date: 2023-12-13 15:00:00
+---
+
+We are excited to announce that we have partnered with Google and [The Forge](https://theforge.dev/) to bring some helpful performance optimizations to our Vulkan mobile backend. This will primarily benefit users targeting Vulkan-capable mobile devices.
```

Comment:
Maybe link Google, just in case (hahahaha) someone doesn't know what it is?

Or more sensibly, if they have some kind of Google/developer/gaming portal, link to that?

Reply:
www.google.com maybe? In all seriousness, I'm not sure that there is an appropriate site to link to in this case. I also feel weird about the asymmetry in linking the Forge but not Google, but I don't think there is an easy solution.

Reply:
https://about.google/ ?